addons/escoria-ui-simplemouse/inventory/inventory_ui.gd

```diff
@@ -3,35 +3,32 @@ extends ESCInventory
 
 # Whether the inventory is visible currently
 var inventory_visible: bool = false
+
+var hovering_inventory_item: bool = false
+
 var _tween: Tween3
 
+@onready var inventory_scene = $panel
+
 func _ready() -> void:
 	super._ready()
-	# Hide inventory by default
-	$FloatingInventory/panel.position.x = \
-		ProjectSettings.get_setting("display/window/size/viewport_width")
 	_tween = Tween3.new(self)
-
-func _on_inventory_button_pressed():
-	if _tween.is_running():
-		return
-	if inventory_visible:
-		hide_inventory()
-	else:
-		show_inventory()
+	# Hide inventory by default
+	hide_inventory()
 
 
 func show_inventory():
 	_tween.stop()
 	_tween.reset()
-	var start_pos = $FloatingInventory/panel.position.x
-	var end_pos = 120
+	var start_pos_y = inventory_scene.position.y
+	var end_pos_y = 706
```

Comment:
I can't recall: Are the screen coordinates listed in this class relative to whatever resolution the game is being played in? Or is this hardcoded, and if the gamedev is using a different resolution, then code like this might make things look weird?

Reply:
Fixed.